cpp/common/src/codingstandards/cpp/deadcode/UnusedFunctions.qll

```diff
@@ -111,6 +113,26 @@ module UnusedFunctions {
     }
   }
 
+  /**
+   * A `MemberFunction` which is either a Default constructor, Destructor
+   * CopyConstructor, CopyAssingmentOperator, MoveConstructor or a
+   * MoveAssignmentOperator
+   */
+  predicate isASpecialMemberFunction(MemberFunction f) {
```

Comment:
There is already a CodeQL class for representing SpecialMemberFunctions:

https://github.com/github/codeql-coding-standards/blob/main/cpp/common/src/codingstandards/cpp/Class.qll#L87-L103

I would recommend using that instead.

Reply:
Sounds good!

Reply:
Fixed in [bd83048](https://github.com/github/codeql-coding-standards/pull/725/commits/bd83048f7d9d9e76a9c7e8a5d6ee0d252947ff83)